react/src/components/AssetsPanel/AssetsPanel.tsx

```diff
@@ -1,19 +1,95 @@
 import React from 'react';
 import styles from './AssetsPanel.module.css';
+import FeatureFileTree from '@hazmapper/components/FeatureFileTree';
+import { FeatureCollection } from '@hazmapper/types';
+import { Button } from '@tacc/core-components';
+import { useFeatures } from '@hazmapper/hooks';
+
+interface DownloadFeaturesButtonProps {
+  projectId: number;
+  isPublic: boolean;
+}
+
+const DownloadFeaturesButton: React.FC<DownloadFeaturesButtonProps> = ({
+  projectId,
+  isPublic,
+}) => {
+  const { isLoading: isDownloading, refetch: triggerDownload } = useFeatures({
+    projectId,
+    isPublic,
+    assetTypes: [], // Empty array to get all features
+    options: {
+      enabled: false, // Only fetch when triggered by user clicking button
+      cacheTime: 0,
+      staleTime: 0,
+      onSuccess: (data: FeatureCollection) => {
+        // Create and trigger download
+        const blob = new Blob([JSON.stringify(data)], {
+          type: 'application/json',
+        });
+
+        const url = window.URL.createObjectURL(blob);
+        const link = document.createElement('a');
+        link.href = url;
+        link.download = `hazmapper.json`;
```

Comment:
Should the name of the download reflect the project name so that the user can differentiate downloads?